### ISSUE TITLE
Version overrided when not duplicated in $options

### ISSUE THREAD

**Issue by MariusJam** [mailjet/mailjet-apiv3-php#254]
Hello,

When you do :

```php
$mj = new \Mailjet\Client(getenv('MJ_APIKEY_PUBLIC'), getenv('MJ_APIKEY_PRIVATE'),true,['version' => 'v3.1']);

/* set body */

$mj->post(Resources::$Email, ['body' => $body], ['timeout' => 5]);
```

The POST request is sent to https://api.mailjet.com/v3/send and not https://api.mailjet.com/v3.1/send as expected. The $options array provided in post() method erases the version set previously.

I have this issue with at least v1.5.5 and v1.5.6 of the Mailjet PHP client.

**Comment by oleksandr-mykhailenko** [comment]
Hello @MariusJam 
Issue was related and published in new version - https://github.com/mailjet/mailjet-apiv3-php/releases/tag/v1.5.7

**Comment by MariusJam** [comment]
Hello @oleksandr-mykhailenko,

Thanks for the quick fix ! The version 1.5.7 resolves the issue indeed.
